assembl/models/preferences.py

```diff
@@ -775,3 +775,15 @@ def get_preference_data_list(self):
     preference_data_key_list = [p["id"] for p in preference_data_list]
     preference_data_key_set = set(preference_data_key_list)
     preference_data = {p["id"]: p for p in preference_data_list}
+
+
+def includeme(config):
+    """Initialize some preference values"""
+    from ..auth.social_auth import get_active_auth_strategies
+    # Note: This is temporary code, will probably create a new scalar type.
+    settings = config.get_settings()
+    # TODO: Give linguistic names to social auth providers.
+    active_strategies = {
+        k: k for k in get_active_auth_strategies(settings)}
+    active_strategies[''] = _("No special authentication")
+    Preferences.preference_data['authorization_server_backend']['scalar_values'] = active_strategies
```

Comment:
Not exactly. This construction from local.ini is the basis. What is read from sqla is overlaid on that. In most cases, that particular value (the preference_data) will not be in the sqla (the preferences are sparse) hence what was built from local.ini will be preserved. But in theory, if we make a change to preferences, we should make a migration to preference_data that could be in the Preference jsons in sqla. I have not done it because it's never been stored yet afaik... a bit fragile, but maximally flexible.